src/db/index.ts

```diff
@@ -1,14 +1,19 @@
-import Dexie, { Table } from 'dexie';
-import { MetadataCache } from './User';
+import Dexie, { Table } from "dexie";
+import { MetadataCache } from "./User";
+import { hexToBech32 } from "../Util";
 
 
 export class SnortDB extends Dexie {
   users!: Table<MetadataCache>;
 
   constructor() {
     super('snortDB');
-    this.version(1).stores({
-      users: '++pubkey, name, display_name, picture, nip05' // Primary key and indexed props
+    this.version(2).stores({
```

Comment:
Do you need to keep the v1 specification?

Reply:
not needed.